client/src/app/app.module.ts

```diff
@@ -3,30 +3,37 @@ import { NgModule } from '@angular/core';
 import { HttpClientModule } from '@angular/common/http';
 import { HttpModule } from '@angular/http';
 import { FormsModule } from '@angular/forms';
-import { StoreModule } from '@ngrx/store';
-import { StoreDevtoolsModule } from '@ngrx/store-devtools';
-import { EffectsModule } from '@ngrx/effects';
 
 import {
-  SocialLoginModule,
   AuthServiceConfig,
+  SocialLoginModule,
   GoogleLoginProvider,
 } from 'angular-6-social-login';
 
-import { SigninComponent } from './signin';
 import { AppComponent } from './app.component';
-import { CardComponent } from './card/card.component';
+import { CardComponent } from './components/card/card.component';
+import { DialogOverviewExampleDialogComponent } from './components/add-user-dialog';
+import { WelcomePageComponent } from './pages/welcome/welcome.component';
+
 import { TimerService } from './services/timer.service';
 import { SocketService } from './services/socket.service';
 import { UserService } from './services/user.service';
-import { DialogOverviewExampleDialogComponent } from './add-user-dialog/add-user-dialog.component';
+import { FightService } from './services';
 
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { DemoMaterialModule } from './material-module';
 import { MatNativeDateModule } from '@angular/material';
-import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-import { FightService } from './services';
-import { reducers } from 'store/store.config';
-import {CardsEffects} from 'store/cards';
+import { FightPageComponent } from './pages/fight/fight.component';
+
+import { RouterModule, Routes } from '@angular/router';
+import { NotFoundPageComponent } from './pages/not-found/not-found.component';
+import { reducers } from './../store';
```

Comment:
relative path

Reply:
probably i fixed that, can you check tsconfig file?